src/PointwiseFunctions/GeneralRelativity/ComputeGhQuantities.hpp

```diff
@@ -129,4 +129,128 @@ tnsr::ii<DataType, SpatialDim, Frame> extrinsic_curvature(
 template <size_t SpatialDim, typename Frame, typename DataType>
 tnsr::ijj<DataType, SpatialDim, Frame> deriv_spatial_metric(
     const tnsr::iaa<DataType, SpatialDim, Frame>& phi) noexcept;
+
+/*!
+ * \ingroup GeneralRelativityGroup
+ * \brief Computes spatial derivatives of the shift vector from
+ *        the generalized harmonic and geometric variables
+ *
+ * \details Let the generalized harmonic spatial derivative variable be
+ * \f$\phi_{iab} = \partial_i \psi_{ab} \f$. The spatial derivatives of shift
+ * vector \f$N^i\f$ are:
+ * \f{align*}
+ *  \psi^{ja} N n^b \phi_{iab} - 2 n^j \partial_i N
```

Comment:
For consistency, the `\phi` and `\pi` should be uppercase `\Phi`, `\Pi`.
also, the lowercase `n`s should be `t`s